### PR TITLE
Update Immich Icon

### LINKED ISSUE #10737: Update: Immich icon
### Brand Name

Immich

### Official Resources for Icon and Color

svg: https://github.com/immich-app/immich/blob/main/design/immich-logo.svg

### Additional Comments

Immich is licensed under the AGPL-3.0 license.
https://immich.app

### PR DESCRIPTION
**Pull request by mani-sh-reddy**
immich-2

**Issue:** closes #10737

**Popularity metric:** [28.9k Github Stars](https://github.com/immich-app/immich)

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description
Updated icon for Immich.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -7266,8 +7266,8 @@
         },
         {
             "title": "Immich",
-            "hex": "4250AF",
```
Comment (uncenter) [inline _data/simple-icons.json:7269]: Where are you getting this new color from? I'm still seeing `#4250AF` as the primary color on their website (in light mode).

File: icons/immich.svg
Comment (uncenter) [inline icons/immich.svg:1]: I'm seeing some differences when I overlay the original icon on top of yours.
Screenshot 2024-03-29 at 16 43 55 (Inkscape)

Comment (uncenter) [reply icons/immich.svg:1]: I was working on updating the icon before you opened this PR, feel free to use https://github.com/uncenter/simple-icons/blob/chore/update-immich/icons/immich.svg.

Comment (mani-sh-reddy) [reply icons/immich.svg:1]: Ah, looks like the source svg is not an exact square (24x23.46)

File: _data/simple-icons.json (hunk 1 above)
Comment (mani-sh-reddy) [reply _data/simple-icons.json:7269]: Got the color straight from the official SVG here: https://github.com/immich-app/immich/blob/25c9b779e4d19bc34551f8b137266a459e0d70e1/design/immich-logo-stacked-light.svg


Screenshot 2024-03-29 at 21 05 05

Comment (uncenter) [reply _data/simple-icons.json:7269]: That's a multi-colored SVG though. I think it's best to stick with their primary color on the website.

Comment (mani-sh-reddy) [reply _data/simple-icons.json:7269]: Reverted to use 4250AF

**Review by uncenter — APPROVED** [review]
Looking good! Using the difference tool reported one difference but it is so small and imperceptible (likely just a rounding/precision issue) that I think we can safely ignore it.

Screenshot 2024-04-22 at 08 37 36 (Inkscape)

**Review by adamrusted — APPROVED** [review]
LGTM, thanks @mani-sh-reddy!

**Pull request merged**